nemo/collections/nlp/models/language_modeling/megatron_base_model.py

```diff
@@ -379,8 +379,11 @@ def _enable_nvidia_optimizations(self):
         # NVIDIA container version check
         nvidia_torch_version = os.getenv('NVIDIA_PYTORCH_VERSION', None)
 
-        # Support DLFW master container
-        if nvidia_torch_version == 'master':
+        def is_official_release_version(nvidia_torch_version):
+            return re.fullmatch("[0-9][0-9]\.[0-9][0-9].*", nvidia_torch_version)  # "YY.MM.*"
```

Comment:
This is currently in the nemo 1.0 path, we'll need something similar for 2.0

Reply:
@thomasdhc Can we unblock this for 1.0 and file a JIRA for the 2.0 code path ?